tests/mutex.rs

```diff
@@ -158,26 +163,61 @@ fn try_unwrap_multiply_referenced() {
 #[test]
 fn with_err() {
     let mtx = Mutex::<i32>::new(-5);
-    let r = current_thread::block_on_all(lazy(|| {
+    let mut rt = current_thread::Runtime::new().unwrap();
+    let r = rt.block_on(lazy(|| {
         let fut = mtx.with(|guard| {
             if *guard > 0 {
                 Ok(*guard)
             } else {
                 Err("Whoops!")
             }
-        });
+        }).unwrap();
         fut.map(|r| assert_eq!(r, Err("Whoops!")))
     }));
+    println!("r is {:?}", r);
```

Comment:
leftover?

Reply:
shit, thank's for noticing that.  I'll fix it, and merge.